_data/settings.yml

```diff
@@ -0,0 +1,321 @@
+experience:
+- {name: Junior UX Designer, icon: 'icon-1.png', date: 'Jan 2007 - Feb 2009', organization: 'WEBEX'}
+- {name: UX & UI Designer, icon: 'icon-2.png', date: 'Mar 2009 - Aug 2014', organization: 'AUGMEDIX'}
+- {name: Senior UI Designer, icon: 'icon-3.png', date: 'Sep 2014 - Present', organization: 'THEMEFISHER'}
+
+
+#-------------------------------
+# Education Section
+
+education-title: Education
+education:
+- {name: Marters in UX Design, date: '2006', organization: 'Masassusets Institute of Technology'}
+- {name: Honours in Fine Arts, date: '2004', organization: 'Harvard University'}
+- {name: Higher Secondary Certificat, date: '2000', organization: 'Cardiff School'}
+- {name: Secondary School Certificate, date: '1998', organization: 'Cardiff School'}
+
+
+#-------------------------------
+# Services Section
+
+features-title: Features
+features:
+- {
+    name: OS Managmenet,
+    icon: ti-package,
+    description: 'Manage Operating system lifecycle using RPM ostree technology and manage the current state of the device.',
+    active: # Add true to select section
+  }
+- {
+    name: Observability,
+    icon: ti-search,
+    description: 'Get any information on the workloads running in the Edgedevice and export it to a central Prometheus instance with limited bandwidth, at the same time export all logs to a central device in case it’s needed.',
+    active: true # Add true to select section
+  }
+
+- {
+    name: Identity aware,
+    icon:  ti-shield,
+    description: 'All devices have their own identity managed by X509 certificates and all communication with the Fleet Management API is secured by  MTLS connection, which also managed the device approval to enrol into the fleet.',
+    active: # Add true to select section
+  }
+
+- {
+    name: Low footprint,
+    icon: ti-wand,
+    description: 'Flotta device is a lightweight daemon that is able to run on disconnected environments or not reliable networks, also managed to limit the control-plane bandwidth.',
+    active: true # Add true to select section
+  }
+- {
+    name: Data Upload,
+    icon: ti-upload,
+    description: 'Upload any data produced by your workload to a centrallized location from all of your devices',
+    active: false # Add true to select section
+  }
+- {
+    name: Ansible integration,
+    icon: ti-plug,
+    description: 'Execute Ansible playbooks on specific subset of Flotta devices and check their execution status',
+    active: true # Add true to select section
+  }
+
+# - {
+#     name: FDO,
+#     icon: ti-vector,
+#     description: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.',
+#     active: true # Add true to select section
+#   }
+
+# - {
+#     name: GIT OPS friendly,
+#     icon: ti-vector,
+#     description: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.',
```

Comment:
We have a [pipeline](https://github.com/project-flotta/examples/tree/main/poc-demo/pipeline) example based on teckton.